src/Adapter/Cache/Clearer/SymfonyCacheClearer.php

```diff
@@ -70,18 +70,13 @@ public function clear()
             // Clear cache
             $input = new ArrayInput([
                 'command' => 'cache:clear',
-                '--no-warmup' => true,
+                '--no-optional-warmers' => true,
```

Comment:
This will ignore this specific warmer : https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Cache/CacheWarmer.php

Is it an issue ?